recipe/meta.yaml

```diff
@@ -16,6 +16,8 @@ source:
     - 0001-conda-Unbundle-libxgboost.-dll-dylib-so.patch
     - 0001-Force-endian-flag-in-cross-compilation-mode.patch  # [arm64 or aarch64 or ppc64le]
     - 0002-Use-static-cudart.patch
+    # xref: https://github.com/dmlc/xgboost/pull/9602
+    - 0003-numpy-ndarray-check.patch
 
 build:
   number: {{ build_number }}
```

Comment:
Also please bump the `build_number` value up top (sorry diff doesn't allow me to comment there to propose the change)

Reply:
Done, I always forget that!

Reply:
No worries. Thanks Dante! 🙏